tests/testthat/test_array_manipulation.r

```diff
@@ -46,9 +46,17 @@ test_that('get.limit.index sets the right indexes', {
   expect_equal(lim_index$end, c(7, 8, 9, 10))  
 })
 
+
 test_that('ellip.limit.vec returns correctly for numerics', {
-  ellip.h <- '\u22EF'
-  
+  # copy of the function repr_matrix_df.r
+  .char_fallback <- function(char, default) {
```

Comment:
didn't work = the test failed even more... (actually I was walking in the dark with this: I've no clue what and why test failed on windows) If you tell me that adding the function on top of the file will still elt it pass on travis, thats fine with me.